app/views/events/index.html.erb

```diff
@@ -11,8 +11,15 @@
 <%= render 'events/aside' %>
 
 <article>
+  <% @upcoming.load %>
```

Comment:
I think the template will look clearer without this line. Probably not worth it for the sake of saving one `count` query.